lua/lvim/utils/hooks.lua

```diff
@@ -68,14 +69,15 @@ function M.run_post_update()
   M.reset_cache()
 
   Log:debug "Syncing core plugins"
-  require("lvim.plugin-loader").sync_core_plugins()
+  require("lvim.plugin-loader").sync_core_plugins(function()
```

Comment:
this is the wrong function, you're probably thinking of  `update_base_lvim()`

Reply:
no I put it there on purpose, because putting `make_changelog` in `update_base_lvim()` or in line 81, displays the window before packer's window and when packer window kicks in it overlaps with changelog window, so showing it after packer is done makes more sense